python/ray/dataframe/dataframe.py

```diff
@@ -1720,9 +1788,24 @@ def from_records(self, data, index=None, exclude=None, columns=None,
             "github.com/ray-project/ray.")
 
     def ge(self, other, axis='columns', level=None):
-        raise NotImplementedError(
-            "To contribute to Pandas on Ray, please visit "
-            "github.com/ray-project/ray.")
+        """Checks element-wise that this is greater than or equal to other.
+
+        Args:
+            other: A DataFrame or Series or scalar to compare to.
+            axis: The axis to perform the gt over.
+            level: The Multilevel index level to apply gt over.
+
+        Returns:
+            A new DataFrame filled with Booleans.
+        """
+        if isinstance(other, DataFrame):
+            return self._inter_df_op_helper(
+                lambda x, y: x.ge(y, axis, level),
+                other, axis, level)
+        else:
+            return self._single_df_op_helper(
+                lambda df: df.ge(other, axis, level),
+                other, axis, level)
```

Comment:
instead of this `if` statement, should we just use `_iter_and_single_df_op_helper`?